src/rpc/server/networkDispatchers.ts

```diff
@@ -33,6 +34,7 @@ export class RequestDispatcher extends Dispatcher<Request, RequestInitializer> i
 
   constructor(scope: DispatcherScope, request: Request) {
```

Comment:
Should we make this constructor private, so that everyone uses RequestDispatcher.from? Same for response.